### DISCUSSION TITLE
How to connect wireUI with Vite/Laravel issue

### DISCUSSION THREAD

**Discussion by Skreng** [wireui/wireui#426] [Q&A]
Hello 
How I should connect WireUI with Vite in newst laravel version? 
Now if I use
```<!DOCTYPE html>
<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <meta name="csrf-token" content="{{ csrf_token() }}">

        <title>{{ config('app.name', 'Laravel') }}</title>

        <!-- Fonts -->
        <link rel="stylesheet" href="https://fonts.bunny.net/css2?family=Nunito:wght@400;600;700&display=swap">
        @livewireStyles
        <wireui:scripts />

        <!-- Scripts -->
        @vite(['resources/css/app.css', 'resources/js/app.js'])

    </head>
    <body class="font-sans antialiased">
        <div class="min-h-screen bg-gray-100">
            @include('layouts.navigation')

            <!-- Page Heading -->
            <header class="bg-white shadow">
                <div class="max-w-7xl mx-auto py-6 px-4 sm:px-6 lg:px-8">
                    {{ $header }}
                </div>
            </header>

            <!-- Page Content -->
            <main>
                {{ $slot }}
            </main>

        </div>
        @livewireScripts
    </body>
</html>

 ```

But every label in the login/register form disappear and no reaction on click login/register button. No error in the console.
   
![obraz](https://user-images.githubusercontent.com/99440602/183478521-5195611a-5f82-4f20-a682-fc57090e74d9.png)
![obraz](https://user-images.githubusercontent.com/99440602/183478612-48bba215-804c-45a1-9d7f-0ed1d1398516.png)
![obraz](https://user-images.githubusercontent.com/99440602/183478873-e480a6ce-b9f0-4e7b-87b3-a9894b38761a.png)
![obraz](https://user-images.githubusercontent.com/99440602/183478996-fe0afc2a-a30a-4d75-8f93-ac3f68542a86.png)

**Comment by ph7jack** [comment]
@Skreng I'm sorry, but I didn't understand your problem.
The wireui will works normally even if using/not using the vite. The vite will process only the project assets.
I recommend creating a separate file to alpinejs, it solves most bugs with alpine
Have you added the wireui content paths into your tailwind CSS config?

**Reply by Skreng** [reply]
Silly thing. I realize that Laravel use login and register forms components <x-input> and <x-button> And the WireUI also use the same component name <x-input> and <x-button> but with different params. Just need to change the Laravel component uses on normal input or button and everything is OK.
